lizard_auth_server/templates/lizard_auth_server/profile.html

```diff
@@ -4,11 +4,12 @@
 {% block subtitle %}{% trans 'Your profile' %}{% endblock %}
 
 {% block container %}
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
a) Indentatie is nog niet consistent: 2 spaties.

b) font-awesome is prima. Waarschijnlijk voor de vuilnisbak? Ik zou hem zelf dan liever in de base.html zien bij de rest van de layout. Kunnen we 'em vaker gebruiken :-)

Reply:
Done